src/virtual-machine.js

```diff
@@ -289,6 +290,16 @@ class VirtualMachine extends EventEmitter {
         }
 
         const validationPromise = new Promise((resolve, reject) => {
+            try {
+                const sb1 = new SB1File(input);
+                const json = sb1.json;
+                json.projectVersion = 2;
+                console.log(json);
+                return resolve([json, sb1.zip]);
```

Comment:
I think there should be a comment here that explains that `sb1.zip` doesn't actually include the `project.json` file, only the sounds and images (in case that is a point of confusion later down the line).